frontend/src/app/conference/conference-detail/conference-detail.component.html

```diff
@@ -0,0 +1,27 @@
+<h2 class="ml-3">{{conference.name}}</h2>
+<div class="ml-3">
+  <button mat-raised-button color="primary"  (click)="interested()" [disabled]="possibleToAttend">Interested</button>
+</div>
+
+<mat-tab-group mat-align-tabs="start" style="height: 600px">
+  <mat-tab label="General Info"><p class="ml-3"> {{conference.generalInfo}}</p></mat-tab>
+  <mat-tab label="Program Committee Members">
+    <mat-list>
+      <mat-list-item *ngFor="let pcMember of conference.pcMembers" class="ml-3" style="width: 600px">
+        <mat-icon mat-list-icon>face</mat-icon>
+        {{pcMember.user.name}}
+      </mat-list-item>
+    </mat-list>
+  </mat-tab>
+  <mat-tab label="Application Details">
+    <div class="ml-3">
+      <p>Conference will begin on {{conference.startTime | date: 'shortDate'}} and end on {{conference.endTime | date: 'shortDate'}}</p>
```

Comment:
please make the dates bold so it looks better :D

Reply:
also, if the deadline is reached, pls display a corresponding messaged that says so